packages/main/src/plugin/kubernetes/kubernetes-client.ts

```diff
@@ -1852,4 +1853,11 @@ export class KubernetesClient {
     }
     return this.contextsStatesDispatcher?.getContextsHealths();
   }
+
+  public getContextsPermissions(): ContextPermission[] {
+    if (!this.contextsStatesDispatcher) {
+      throw new Error('contextsStatesDispatcher is undefined. This should not happen in Kubernetes experimental');
+    }
+    return this.contextsStatesDispatcher?.getContextsPermissions();
```

Comment:
TypeScript is not able to infer that it's always here ?
```suggestion
    return this.contextsStatesDispatcher.getContextsPermissions();
```